ImageFeed2/Auth/ProtocolWebViewViewControllerDelegate.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ProtocolWebViewViewControllerDelegate.swift
+//  ImageFeed2
+//
+//  Created by Аделия Исхакова on 19.04.2023.
+//
+
+import Foundation
+
+protocol ProtocolWebViewViewControllerDelegate: AnyObject {
```

Comment:
Некорретный нейминг. Ключевого слова делегат уже достаточно, чтобы понять, что это протокол. 
```suggestion
protocol WebViewViewControllerDelegate: AnyObject {
```
в других случаях, когда мы не используем Delegate в неминге, но нам нужен признак протокола, мы добавляем это название в конец (WebViewViewControllerProtocol), а не в начало